source/_integrations/comelit.markdown

```diff
@@ -37,6 +37,34 @@ There is support for the following platform types within Home Assistant:
 
 {% include integrations/config_flow.md %}
 
+{% configuration_basic %}
+  host:
+    description: The IP address of the Comelit SmartHome device.
+  port:
+    description: The TCP port of the Comelit SmartHome device.
+  pin:
+    description: The PIN of the Comelit SmartHome device.
+  type:
+    description: The type the Comelit SmartHome device.
+    keys:
+      bridge:
+        description: Comelit Serial Bridge.
+      vedo:
+        description: Comelit VEDO System.
+{% endconfiguration_basic %}
+
+
+## Data updates
+
+This integration fetches data from the device every 5 seconds by default.
```

Comment:
```suggestion
This integration {% term polling polls %} data from the device every 5 seconds by default.
```
Replacing fetch by poll because we have a glossary entry for it :-D